packages/developer-portal/src/components/analytics/costs/services-table.tsx

```diff
@@ -17,15 +17,14 @@ export interface ServicesTableProps {
 }
 
 export const ServicesTable: FC<ServicesTableProps> = ({ billing }) => {
-  const services = billing?.services ?? []
+  const services = billing?.services?.filter((service) => service.name !== 'API Requests') ?? []
 
   return services.length ? (
     <Table
       className={elMb7}
       rows={
         services
           .map(({ name, amount, cost, itemCount, items }) => {
-            if (name === 'API Requests') return
```

Comment:
this adds an additional loop. Backend I would say we could mitigate some milliseconds by preventing it but your call :) not a big issue.

Reply:
As discussed, only like 5 items absolute max